scripts/partition_manager_report.py

```diff
@@ -86,7 +86,10 @@ def main():
 
     for i in args.input:
         fn = path.basename(i)
-        domain_name = fn[fn.index("partitions_") + len("partitions_"):fn.index(".yml")]
+        if '_' in fn:
```

Comment:
This means someone cannot create a `<company>_mcuboot` partition/child image ?

Is this limitation handle elsewhere in the build system.

Reply:
The `partitions.yml` file is named by the build system, and is not user configurable unless they modify our cmake code.
The user specified partitions are inside the file, not part of the name.

Reply:
but if someone names a partition `my_partitions_app` and creates a `my_partitions_app.yml` file, then that name will propagate through the build system and enter the wrong `if` statement here, right ?

Reply:
The partitions are not reflected in the name of this file.
The name of this file is static and defined by `partiton_manager.cmake`. 
So to create `my_partitions_app.yml` they would have to modify the cmake code which generates the file.

Anyway, I can add the domain list as an argument, so that we don't depend on a simple underscore in the name, to be more future proof.

Reply:
no, think it is fine then.
Just wanted to be sure there would be no errors in due to specific naming, as looking for a sing `_` looked vulnerable.

Thanks for clarifying.